Gruntfile.js

```diff
@@ -67,10 +67,11 @@ module.exports = function (grunt) {
       'softwerkskammer/build/stylesheets/less/agora.less'
     ]
   };
+  const integrationTestCoverageDataFolder =  'softwerkskammer/coverageIntegration';
   grunt.initConfig({
     clean: {
       build: ['softwerkskammer/build', 'softwerkskammer/frontendtests/fixtures/*.html'],
-      coverage: ['softwerkskammer/coverage', 'softwerkskammer/coverageWithDB', 'softwerkskammer/karma-coverage'],
+      coverage: ['softwerkskammer/coverage', integrationTestCoverageDataFolder, 'softwerkskammer/karma-coverage'],
```

Comment:
Der Pfad wird an zwei Stellen verwendet und ich hatte ihn ein paarmal geändert, daher war das nützlich. Der Rule Of Three folgend könnten wir es aber wieder inlinen.